src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -328,8 +328,8 @@ def is_device(dev):
         return False
     # use lsblk first, fall back to using stat
     TYPE = lsblk(dev).get('TYPE')
-    if TYPE:
-        return TYPE == 'disk'
+    if TYPE == 'disk' or TYPE.startswith(('raid')):
```

Comment:
if ``TYPE`` is ``None`` then the ``TYPE.startswith(('raid'))`` with raise an ``AttributeError``. You'll need to keep the ``if TYPE:`` before this line.